src/System.Data.SqlClient/src/System/Data/SqlClient/SqlCommand.cs

```diff
@@ -223,6 +223,23 @@ public SqlCommand(string cmdText, SqlConnection connection, SqlTransaction trans
             Transaction = transaction;
         }
 
+        private SqlCommand(SqlCommand from) : this()
+        {
+            CommandText = from.CommandText;
+            CommandTimeout = from.CommandTimeout;
+            CommandType = from.CommandType;
+            Connection = from.Connection;
+            DesignTimeVisible = from.DesignTimeVisible;
+            Transaction = from.Transaction;
+            UpdatedRowSource = from.UpdatedRowSource;
+
+            SqlParameterCollection parameters = Parameters;
+            foreach (object parameter in from.Parameters)
+            {
+                parameters.Add((parameter is ICloneable) ? (parameter as ICloneable).Clone() : parameter);
```

Comment:
@stephentoub Today I tried building from command line with VS Dev Command Prompt 2017 and by building inside VS 2017 as well after setting up VS 2017 using the instructions at https://github.com/dotnet/corefx/blob/master/Documentation/building/windows-instructions.md .
I couldn't get the compiler to stop complaining. I would like to go ahead without addressing this comment as it is taking up more time compared to the benefits the change offers.